src/ert/gui/tools/workflows/run_workflow_widget.py

```diff
@@ -101,17 +101,17 @@ def cancelWorkflow(self) -> None:
 
             if cancel == QMessageBox.Yes:
                 self._workflow_runner.cancel()
-                self._running_workflow_dialog.disableCloseButton()
+                if self._running_workflow_dialog is not None:
+                    self._running_workflow_dialog.disableCloseButton()
```

Comment:
More compact?
```
if self._running_workflow_dialog:
    self._running_workflow_dialog.disableCloseButton()
```

Reply:
Maybe, but I think its best to avoid messing with `__bool__` for now.